src/components/PlayerView/playerViewConfig.ts

```diff
@@ -19,6 +19,14 @@ export interface PlayerViewConfig {
    * Provides options to configure Picture in Picture playback.
    */
   pictureInPictureConfig?: PictureInPictureConfig;
+
+  /**
+   * When set to `true`, the first frame of the main content will not be rendered before playback starts. Default is `false`.
+   * This configuration has no effect for the ${@link UserInterfaceType.Subtitle} on iOS/tvOS.
```

Comment:
I think there is an extra `$` there
```suggestion
   * This configuration has no effect for the {@link UserInterfaceType.Subtitle} on iOS/tvOS.
```